test/JHipsterNetSampleApplication.Test/Web/Rest/OperationResourceIntTest.cs

```diff
@@ -120,6 +120,38 @@ public async Task CreateOperationWithExistingReferencedEntity()
             testOperation.BankAccount.Should().Be(bankAccount);
         }
 
+        [Fact]
+        public async Task CreateOperationWithManyToManyAssociation()
```

Comment:
Can you test the other side of the navigation too.
Currently
- An object `_operation` exists
- You create a label . and persist it
- Add the label to the `_operation` object and persist it.
- Validation the operation
To add
- Retrieve the label from the DB including the operations
_ Verify the associated operation is the one tested above